discovery-client/src/main/java/io/micronaut/discovery/consul/client/v1/KeyValue.java

```diff
@@ -32,19 +32,28 @@
 @Serdeable
 @ReflectiveAccess
 public class KeyValue {
-    private String key;
-    private String value;
+    private final Integer modifyIndex;
```

Comment:
also adding the `modifyIndex` in the returned item, so it can be used for watching its changes